### DISCUSSION TITLE
The POST method is not supported for route admin/tasks/post. Supported methods: GET, HEAD.

### DISCUSSION THREAD

**Discussion by Evinny** [orchidsoftware/platform#2622] [Q&A]
hi, im new to laravel, and even newer to laravel orchid, and im following the docs to learn it, but ive got myself stuck in this problem.

first, i make my screen, then its route, make it appear on the menu, then the fields, but when i try to save the fields (following the docs on orchid page) i get the error on the title, and if it was normal laravel, i would know how to fix it, but the route set in the platform.php file doesn't seem to let me choose its type.

this is the code inside the tasksscreen file: 
![image](https://github.com/orchidsoftware/platform/assets/114532225/e9bae249-e257-48fb-bd6a-657891199a01)

maybe it is something very dumb and im making fun of myself here, but at least i will be getting closer to the laravel orchid community

since now, i already thank yall

**Comment by tabuna** [comment]
Hi, there is most likely a conflict in route definition. This happens because Laravel chooses the very first route that meets the conditions.

You can see more details here: https://orchid.software/en/docs/screens/#routing-screens
